### PR TITLE
Show videos in media overview

### LINKED ISSUE #5941: Inconsistency: When watching a video and pressing the gallery button, the video is not in the list of "all pictures"
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
Thanks for adding support to play videos in-app. When playing videos of a conversation in-app, the button to go to the gallery ("all pictures") is visible. After pressing the button one can see all the pictures of the conversation. The video, from where I was directed to the gallery, is not visible, though.

### Steps to reproduce
- go to a conversation
- open a video
- press the gallery button ("all pictures")

**Actual result:** I see all pictures of the conversation.
**Expected result:** Two possibilities:
1. I can see the video in the gallery view as well, since the video is where I come from. (Renaming of the string "all pictures" to "all pictures and videos" or similar)
2. There's no gallery button when watching the video.

### Device info
**Signal version:** 3.24.1

### PR DESCRIPTION
**Pull request by haffenloher**
### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Android 6.0 emulator
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description
Fixes #5941

// FREEBIE

### CODE REVIEW

**Comment by FeuRenard** [comment]
Tested with Android 4.4.4, Screenshot below.
device-2017-02-04-105351

**Comment by FeuRenard** [comment]
Did you think about putting "media" instead of "images and videos" in the strings?

**Comment by haffenloher** [comment]
I used "images and videos" because it doesn't include audio, but it's a bit long. In your screen shot, it pushes the recipient's name completely off the screen. Maybe "media" is the better choice here indeed

**Comment by haffenloher** [comment]
Still not 100% sure about this, but I pushed a commit renaming "All images and videos" to "All media" for now.

File: src/org/thoughtcrime/securesms/MediaOverviewActivity.java [hunk 1]
```diff
@@ -240,7 +240,7 @@ public ThreadMediaLoader(Context context, long threadId) {
 
     @Override
     public Cursor getCursor() {
-      return DatabaseFactory.getImageDatabase(getContext()).getImagesForThread(threadId);
+      return DatabaseFactory.getImageDatabase(getContext()).getMediaForThread(threadId);
```
Comment (FeuRenard) [inline src/org/thoughtcrime/securesms/MediaOverviewActivity.java]: getImageDatabase(...) -> getMediaDatabase(...) in DatabaseFactory

Comment (haffenloher) [inline src/org/thoughtcrime/securesms/MediaOverviewActivity.java]: oops, thanks for pointing that out! Also found a string saying "No images" that I had missed. Fixed now.

**Comment by moxie0** [comment]
huh looks simple! have you tested on devices that generate video thumbnails instead of the placeholder?

**Comment by haffenloher** [comment]
not on a physical device, but in a 6.0 emulator. It correctly shows the video thumbnail with a play button.

**Comment by moxie0** [comment]
in 3.30.0

**Pull request closed without merging**